### PR TITLE
handbook: New page for creating screenshots, GIFs, and videos

### PR DESCRIPTION
**Pull request by ryan-blunden**

### CODE REVIEW

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Adding a screenshot or screen recording (GIF and video)
+
+> Note: This guide presumes macOS
+
+## TL;DR
+
+- Favor capturing only the part of the screen that’s required
+- Keep GIFs 15 seconds or less
+- GIFs are typically great for short "check this out" recordings, where videos are better at "how-to" or promotional style content
+- Keep GIF and images as small as possible with no retina sizes (just 1 x)
```
Comment (christinaforney) [inline handbook/marketing/adding_screenshots_screen_recording.md:10]: How do you know if it is retina sized? Is there a setting to look for or way to check for this?

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+- If doing a full-screen recording, 1280x720 (720p) will produce easy to read results at default settings.
+- If you’re producing visual content regularly, invest in paid apps (you can expense them too).
+- Chose CleanShot if just doing screenshots and GIFs.
+- Add ScreenFlow to your list if you plan on doing videos frequently, as you will appreciate editing and clip speed-up capabilities.
```
Comment (christinaforney) [inline handbook/marketing/adding_screenshots_screen_recording.md:14]:
```suggestion
   - Chose CleanShot if just doing screenshots and GIFs.
   - Add ScreenFlow to your list if you plan on doing videos frequently, as you will appreciate editing and clip speed-up capabilities.
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+- Make it as short and simple as possible, but not so short, it’s hard to know what you’re demonstrating
+- Only capture the part of the screen that is necessary. If the output is video, then aim for 16:9 ratio for YouTube
+- If the entire screen needs to be captured, then your resolution should be 1280x720 (720p), as that will make on-screen elements easy to read
+- Decide whether it should be standalone, or if it only makes sense when embedded with content
+- Ideally, GIFs should less than 15-20 seconds in length, otherwise, it’s most likely better as a video
+- Invest in learning a tool such as ScreenFlow if you need to annotate and/or edit your recordings
+- Does your GIF or video make sense without an accompanying explanation? If not, then a VoiceOver or text annotations can help a lot
+
+Like most things, practice makes perfect, but if you’re wanting to get some advice before taking the plunge, Ryan, Thorsten, and Quinn (sqs), have all created screencasts and can help you get started, so just reach out on Slack.
+
```
Comment (christinaforney) [inline handbook/marketing/adding_screenshots_screen_recording.md:70]: A brief section on audio would be really helpful (I've just done it at the same time, but then can't do the nice speed-ups in different sections). 

```suggestion

### Adding audio or voice-overs

TODO

```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+## Embedding GIFs
```
Comment (christinaforney) [inline handbook/marketing/adding_screenshots_screen_recording.md:71]: How do you add a nice intro page and ending page? How do we prefer to title these?

```suggestion

## Adding start/end screens to videos

TODO

## Embedding GIFs
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+3. Add a title that has key words someone is likely to use in a search
+4. Add a 1-2 sentence description of your video that contains a call-to-action for the user to learn more, activate the feature, or at least, install Sourcegraph. See other Sourcegraph videos that may have description content you can re-use
+5. For thumbnail, either select the best option presented or generate your own by viewing the video fullscreen, taking a screenshot, then uploading it
+6. Select "No, it's not made for kids"
+7. Expand ‘MORE OPTIONS" to add the tags: "sourcegraph", "codesearch", "universalcodesearch"
+8. Click NEXT, then NEXT again
+9. Under Visibility, select **Unlisted** if the video only makes sense if embedded in content, or **Public**. If you think the video would benefit from approval first, leave set as **Unlisted** or **Draft**.
+10. Click **SAVE**
+
+Then give the marketing team the heads0up that a new video has been published, letting them know what visibility state the video is currently in.
```
Comment (christinaforney) [inline handbook/marketing/adding_screenshots_screen_recording.md:106]:
```suggestion
Then give the marketing team the heads-up that a new video has been published, letting them know what visibility state the video is currently in.
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+- [ScreenFlow](https://www.telestream.net/screenflow/overview.htm)<br/>
+Simply the best screen recording, editing, and production tool available. Designed primarily for video work (e.g. editing) but can export to GIF.
+
+### Screen recording tips
+
+Recording a great screencast is a large topic, but here are some essentials to keep in mind:
+
+- Ensure it has an easy to demonstrate and clear purpose, e.g. show smart search field autocomplete, syntax highlighting, and error handling
+- Make it as short and simple as possible, but not so short, it’s hard to know what you’re demonstrating
```
Comment (aileenrose) [inline handbook/marketing/adding_screenshots_screen_recording.md:61]:
```suggestion
- Make it as short and simple as possible, but enough to show what you're demonstrating
```

File: handbook/marketing/adding_screenshots_screen_recording.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+- Only capture the part of the screen that is necessary. If the output is video, then aim for 16:9 ratio for YouTube
+- If the entire screen needs to be captured, then your resolution should be 1280x720 (720p), as that will make on-screen elements easy to read
+- Decide whether it should be standalone, or if it only makes sense when embedded with content
+- Ideally, GIFs should less than 15-20 seconds in length, otherwise, it’s most likely better as a video
+- Invest in learning a tool such as ScreenFlow if you need to annotate and/or edit your recordings
+- Does your GIF or video make sense without an accompanying explanation? If not, then a VoiceOver or text annotations can help a lot
```
Comment (aileenrose) [inline handbook/marketing/adding_screenshots_screen_recording.md:67]:
```suggestion
- Does your GIF or video make sense without an accompanying explanation? If not, then a voice-over or text annotations can help a lot
```

File: handbook/marketing/adding_screenshots_screen_recording.md (hunk 1 above)
Comment (ryan-blunden) [reply handbook/marketing/adding_screenshots_screen_recording.md]: Added explanation "(ratio of image size to intended display size is 1:1)"

**Comment by ryan-blunden** [comment]
@christinaforney Also added content for voice-overs and start/end video screens.

**Pull request merged**